core/src/main/java/com/sequenceiq/cloudbreak/service/cluster/EmbeddedDatabaseService.java

```diff
@@ -76,13 +75,12 @@ public boolean isAttachedDiskForEmbeddedDatabaseCreated(ClusterView cluster, Opt
     }
 
     public boolean isSslEnforcementForEmbeddedDatabaseEnabled(StackView stackView, ClusterView clusterView) {
-        String accountId = Crn.safeFromString(stackView.getResourceCrn()).getAccountId();
         StackType stackType = stackView.getType();
         boolean sslEnforcementEnabled = false;
         if (stackType == StackType.DATALAKE) {
-            sslEnforcementEnabled = entitlementService.databaseWireEncryptionEnabled(accountId);
+            sslEnforcementEnabled = true;
         } else if (stackType == StackType.WORKLOAD) {
-            sslEnforcementEnabled = entitlementService.databaseWireEncryptionDatahubEnabled(accountId);
+            sslEnforcementEnabled = true;
         }
```

Comment:
This could be now reduced to:
```
boolean sslEnforcementEnabled = stackType == StackType.DATALAKE || stackType == StackType.WORKLOAD;
```

The only justification for keeping this flag is to handle other `StackType` constants (`LEGACY` and `TEMPLATE`).